### PR TITLE
New internal functions

### PR DESCRIPTION
**Pull request by Ashvin-Ranjan**
We need to make it so that internal functions can take in `any` so we don't need to write individual functions for everything.

### CODE REVIEW

**Comment by SheepTester** [comment]
We could add generics

**Comment by Ashvin-Ranjan** [comment]
Yeah, let's do that in this branch too, I guess you can work on that, and ill work and new internal functions.

File: python/runner.n [hunk 1]
```diff
@@ -0,0 +1,3 @@
+print "test"
+let eeeeee:int = 0
```
Comment (SheepTester) [inline python/runner.n:2]: We should have a way of explicitly declaring what should be exported. I agree with Rust that things should be private by default. I quite prefer simply tagging `export`/`public`/`pub` to something than listing all the exports at the end/top of the file, so perhaps

```js
export let eeeeee: int = 0
public let eeeeee: int = 0
pub let eeeeee: int = 0
```

Go makes capital variable names exported, but I'm not a fan of that

File: python/run.n [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import fek
 import SystemIO
+imp runner
```
Comment (SheepTester) [inline python/run.n:3]: How would one import files with odd names, or files in folders or in parent folders? How would one import libraries?

I quite like Deno's module system:

```ts
import { export1, export2 } from 'https://deno.land/x/some-module/mod.ts'
import * as namespace from './folder/file-name.ts'
```

This way, the file name can be independent of the code.

File: python/syntax.lark [hunk 3]
```diff
@@ -52,6 +54,8 @@ listdef: LIST "[" NAME "]"
 tupleval: "(" value ("," value)* ")"
 listval: "[" (value ("," value)*)? "]"
 recordval: "{" (record_entry (";" | "\n"))* record_entry (";" | "\n")? "}"
+imported_value: NAME "?" NAME
```
Comment (SheepTester) [inline python/syntax.lark:57]: Also maybe we should make all imports records. By merging imports and variables, we can avoid confusing name clashes that currently are allowed, like

```js
import whatever
let whatever = { a: 3; b: 4 }
```

**Comment by SheepTester** [comment]
How did you commit something from `__pycache__`? https://github.com/nbuilding/N-lang/pull/23/files/5d51a2e5efc011e8977e9a4c217357a1cbb6eed9..a3dddeb7b50ba6f4ae24df9d99bacca37c5063b3#diff-a7608e0f32a65611d198e7406121c7a392052ccd8a5ff73b56d05604d05c8987

image

**Comment by SheepTester** [comment]
When'll we merge this?

**Pull request merged**